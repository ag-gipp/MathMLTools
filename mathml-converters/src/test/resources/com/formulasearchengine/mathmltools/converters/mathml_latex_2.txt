<formula xmlns="http://www.tei-c.org/ns/1.0"
         n="(1)"
         notation="TeX"
         rend="display"
         xml:id="M1">\documentclass[10pt]{article}
    \usepackage{amsmath}
    \usepackage{wasysym}
    \usepackage{amsfonts}
    \usepackage{amssymb}
    \usepackage{amsbsy}
    \usepackage{mathrsfs}
    \usepackage{pmc}
    \usepackage[Euler]{upgreek}
    \pagestyle{empty}

    \oddsidemargin -1.0in

    \begin{document}
    \begin{equation*}J(t)=a(t/t_{0})^{{\mathrm{b}}},\end{equation*}\end{document}</formula>