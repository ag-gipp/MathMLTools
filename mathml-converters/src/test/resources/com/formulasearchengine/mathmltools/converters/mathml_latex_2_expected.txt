exception expected in the current implementation